Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -2323,3 +2325,26 @@ linkRCWithDriverForFleet merchantShortId opCity fleetOwnerId req = do
     driverRCAssoc <- makeRCAssociation driver.merchantId driver.merchantOperatingCityId driver.id rc.id (convertTextToUTC (Just "2099-12-12"))
     QRCAssociation.create driverRCAssoc
   return Success
+
+postDriverClearFee :: (ShortId DM.Merchant -> Context.City -> Maybe Text -> API.Types.ProviderPlatform.Management.Driver.ClearDriverFeeReq -> Flow APISuccess)
+postDriverClearFee _merchantShortId _opCity a1 req = do
```

Comment:
could you rename a1 to something more clear? by the way in the next PR I've added change to DSL for human readable params